requirements.txt

```diff
@@ -1,10 +1,9 @@
 Django==1.9.2
 dj-database-url==0.4.0
-django-allauth==0.24.1
+-e git://github.com/pennersr/django-allauth.git@6fdade4f65fb86f8d085c68fb3950799e7cf9e05#egg=django_allauth
```

Comment:
Won't be able to merge this until django-allauth makes an official release out of it. Only going to deploy official releases.